diff -ruN baltech_orig/baltech_api/c/CMakeLists.txt baltech/baltech_api/c/CMakeLists.txt
--- baltech_orig/baltech_api/c/CMakeLists.txt	2019-12-11 18:59:04.000000000 +0100
+++ baltech/baltech_api/c/CMakeLists.txt	2020-11-24 20:42:22.608801892 +0100
@@ -2,6 +2,7 @@
 project (brp_lib C)
 
 
+include_directories(../../brp_lib/inc)
 add_library(baltech_api STATIC baltech_api.c)
 target_include_directories(baltech_api INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(baltech_api brp_lib)
diff -ruN baltech_orig/brp_lib/CMakeLists.txt baltech/brp_lib/CMakeLists.txt
--- baltech_orig/brp_lib/CMakeLists.txt	2020-08-07 16:48:56.000000000 +0200
+++ baltech/brp_lib/CMakeLists.txt	2020-11-24 20:42:27.722845146 +0100
@@ -1,5 +1,4 @@
-cmake_minimum_required (VERSION 3.15)
-cmake_policy(SET CMP0091 NEW)
+cmake_minimum_required (VERSION 3.13)
 project (brp_lib C)
 
 
@@ -50,4 +49,4 @@
 endif()
 
 
-add_subdirectory(thirdparty)
\ No newline at end of file
+add_subdirectory(thirdparty)
diff -ruN baltech_orig/CMakeLists.txt baltech/CMakeLists.txt
--- baltech_orig/CMakeLists.txt	2020-08-07 16:49:06.000000000 +0200
+++ baltech/CMakeLists.txt	2020-11-24 20:42:22.608801892 +0100
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.15)
+cmake_minimum_required (VERSION 3.13)
 cmake_policy(SET CMP0091 NEW)
 project (baltech_sdk C)
 
